example/lib/main.dart

```diff
@@ -174,7 +251,7 @@ class TasksTextField extends StatelessWidget {
     if (message.trim().isEmpty) {
       _textEditingController.clear();
     } else {
-      final task = Task(null, message, DateTime.now());
+      final task = Task(null, false, message, DateTime.now(), TaskType.open);
```

Comment:
Yes, I think it would be better to add a factory constructor with optional parameters that would be predefined by default values.

Reply:
> Not your fault, but maybe a named constructor would make this more readable. Specially the null and false.

Yes, I think it would be better to add a factory constructor with optional parameters that would be predefined by default values.